src/data/nat/factorization.lean

```diff
@@ -203,6 +203,11 @@ lemma prime_pow_dvd_iff_le_factorization (p k n : ℕ) (pp : prime p) (hn : n 
   p ^ k ∣ n ↔ k ≤ n.factorization p :=
 by rw [←factorization_le_iff_dvd (pow_pos pp.pos k).ne' hn, pp.factorization_pow, single_le_iff]
 
+lemma prime_pow_dvd_multiplicity_iff (p k n : ℕ) (hp : nat.prime p) (hpos : n ≠ 0) :
```

Comment:
Often for these `iff`-lemmas, implicit arguments are preferred. Would it make sense to replace `(p k n : ℕ)` with `{p k n : ℕ}` in this lemma and the lemma above?